app/src/main/res/layout/tracking_protection_category.xml

```diff
@@ -52,7 +37,6 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toEndOf="@id/trackingProtectionCategoryIcon"
-        app:layout_constraintTop_toBottomOf="@+id/trackingProtectionCategoryTitle"
-        tools:text="@tools:sample/lorem/random" />
+        app:layout_constraintStart_toStartOf="parent"
```

Comment:
This probably changes the look of this screen right? ie the text is much more left aligned than it used to be? I'm fine with landing these changes but could you follow up with UX to verify how this should be without icons?

Reply:
Shoot! good catch